src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -194,20 +217,18 @@ public ObjectResult(object value)
             return selectedFormatter;
         }
 
-        private static MediaTypeHeaderValue[] SortMediaTypeHeaderValues
-                                                    (IEnumerable<MediaTypeHeaderValue> headerValues)
+        private bool HasWildCardMediaAndSubMediaType(IEnumerable<MediaTypeHeaderValue> mediaTypes)
```

Comment:
probably do not need a separate method .. upto you.
If you decide to leave the method, have it as static